### ISSUE TITLE
QRF deadlock- Troops dropped into harbour basin

### ISSUE THREAD

**Issue by CurePindal** [A3Antistasi/antistasiofficial#257]
Two times we had the situation that we captured the Kavala Harbour and 3 QRF choppers dropped their infantry into the harbour basin. The choppers have been shot down but the infantry was just swimming out there in the water, doing nothing. So we ignored it.
This somehow deadlocked all further QRFs. Whatever location we attacked, we got no more QRFs until we restarted the server.
Maybe the choppers should do a check if they drop the troops over land to avoid this?

**Comment by StefArma** [comment]
Indeed it need a check on landing position, we have it on NATO air assault... the problem is that sometime wind move parachutes away... so probably there should be also a wind check...

**Comment by zalexki** [comment]
Yes either check if landing is above water or teleport unit on closest ground.
